src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -124,15 +147,25 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 		if (!SKRIPT_COMMAND_HELP.test(sender, args))
 			return true;
 
+		boolean isReload = label.equalsIgnoreCase("reload");
```

Comment:
does this work? i believe you need to check `args[0].equalsIgnoreCase("reload")`